elasticapm/traces.py

```diff
@@ -709,13 +744,52 @@ def set_transaction_name(name, override=True):
 
 
 def set_transaction_result(result, override=True):
+    """
+    Sets the result of the transaction. The result could be e.g. the HTTP status class (e.g "HTTP 5xx") for
+    HTTP requests, or "success"/"fail" for background tasks.
+
+    :param name: the name of the transaction
+    :param override: if set to False, the name is only set if no name has been set before
+    :return: None
+    """
+
     transaction = execution_context.get_transaction()
     if not transaction:
         return
     if transaction.result is None or override:
         transaction.result = result
 
 
+def set_transaction_success():
```

Comment:
Are you planning to add an API for spans too?

Reply:
Not in this form, no. Spans that are created by auto-instrumentation aren't accessible to the user (at least not in a documented way). Manually created spans have the `set_success()` and `set_failure()` methods.